### PR TITLE
ci: maybe fix issue label automation

### PR DESCRIPTION
**Pull request by rootulp**
Attempts to close https://github.com/celestiaorg/celestia-app/issues/2262

### CODE REVIEW

File: .github/workflows/issue-label-automation.yml [hunk 1]
```diff
@@ -10,9 +10,12 @@ jobs:
     permissions:
       issues: write
       pull-requests: write
+    env:
+      IS_BOT: ${{ github.actor == 'dependabot[bot]' || github.actor == 'mergify[bot]' }}
+      IS_HUMAN: ${{ github.actor != 'dependabot[bot]' && github.actor != 'mergify[bot]' }}
```
Comment (SweeXordious) [inline .github/workflows/issue-label-automation.yml:15]: [optional]
```suggestion
      IS_HUMAN: ${{ !env.IS_BOT }}
```

**Review by SweeXordious — APPROVED** [review]
utACK

**Review by evan-forbes — APPROVED** [review]
utACK

**Comment by rootulp** [comment]
Dang this didn't work. https://github.com/celestiaorg/celestia-app/issues/2281

**Pull request merged**